adapters/socketio_berkeley.c

```diff
@@ -638,8 +638,12 @@ int socketio_setoption(CONCRETE_IO_HANDLE socket_io, const char* optionName, con
         }
         else if (strcmp(optionName, "tcp_keepalive_time") == 0)
         {
+#if TARGET_OS_IPHONE
+            result = -1;
+#else
             result = setsockopt(socket_io_instance->socket, SOL_TCP, TCP_KEEPIDLE, value, sizeof(int));
```

Comment:
>SOL_TCP [](start = 60, length = 7)

Apparently this should be IPPROTO_TCP for the level. for both calls (here and in line 650).